inc/editor/blocks/section/class-advanced-column-css.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace ThemeIsle\BlockCSS;
+
+use ThemeIsle\BlockCSS;
+
+/**
+ * Class Advanced_Column_CSS
+ */
+class Advanced_Column_CSS extends BlockCSS {
+
+	/**
+	 * The namespace under which the blocks are registered.
+	 *
+	 * @var string
+	 */
+	public $block_prefix = 'advanced-column';
+
+	/**
+	 * Constructor function for the module.
+	 *
+	 * @method __construct
+	 */
+	public function __construct() {
+		parent::__construct();
+	}
+
+	/**
+	 * Generate Advanced Column CSS
+	 * 
+	 * @since   1.2.5
+	 * @access  public
+	 */
+	public function render_css( $block ) {
+		$attr = $block['attrs'];
+		$style = '';
+
+		if ( isset( $attr['id'] ) ) {
+			$style .= '#' . $attr['id'] . ' {' . "\n";
+			if ( 'linked' === $this->get_attr_value( ( isset( $attr['paddingType'] ) ? $attr['paddingType'] : null ), 'linked' ) ) {
+				$style .= '	padding: ' . $this->get_attr_value( ( isset( $attr['padding'] ) ? $attr['padding'] : null ), 20 ) . 'px;' . "\n";
+			}
+
+			if ( 'unlinked' === $this->get_attr_value( ( isset( $attr['paddingType'] ) ? $attr['paddingType'] : null ), 'linked' ) ) {
+				$style .= '	padding-top: ' . $this->get_attr_value( ( isset( $attr['paddingTop'] ) ? $attr['paddingTop'] : null ), 20 ) . 'px;' . "\n";
```

Comment:
There is a lot of repeated code, following this pattern isset( $attr['paddingTop'] ) ? $attr['paddingTop'] : null `

You can use `wp_parse_args` to define a default value for the `attr`, something like: 
```
$attr = wp_parse_args($args, [
'paddingTop'=>null,
'paddingBottom' => null,
...
]);
```

Reply:
This we can abstract in the next version and improve as it will delay the release.